controllers/prep_handlers.go

```diff
@@ -444,6 +444,14 @@ func (r *ImageBasedUpgradeReconciler) handlePrep(ctx context.Context, ibu *lcav1
 
 	switch {
 	case !r.PrepTask.Active:
+		r.Log.Info("Running health check for Prep")
+		if err := CheckHealth(ctx, r.NoncachedClient, r.Log); err != nil {
+			msg := "system must be stable before Prep stage can continue"
+			r.Log.Error(err, msg)
+			utils.SetPrepStatusFailed(ibu, fmt.Sprintf("%s: %s", msg, err.Error()))
+			return doNotRequeue(), nil
```

Comment:
These should all be requeueWithHealthCheckInterval, shouldn't they?

Reply:
I was thinking "stop everything if health is bad" but yeah changing to requeueWithHealthCheckInterval works too

Reply:
Updated to requeue